ecosystem/sep-0031.md

```diff
@@ -189,7 +215,9 @@ The response should be a JSON object like:
 }
 ```
 
-The JSON object contains an entry for each currency that the anchor supports for receiving and completing a direct payment.
+The JSON object contains an entry for each Stellar asset that the Receiving Anchor supports receiving from the Sending Anchor. 
+
+If the Receiving Anchor supports [SEP-38](sep-0038.md), the Sending Client can check the [`SEP-38 GET /prices`](sep-0038.md#get-prices) endpoint to determine which off-chain assets the Receiving Anchor can deliver to recipients in exchange for an asset listed in the [`SEP-31 GET /info` response](#get-info). If the Receiving Anchor does not provide exchange rates for an asset, it is assumed that the Stellar asset can be exchanged 1-for-1 with the corresponding off-chain asset.
```

Comment:
> If the Receiving Anchor does not provide exchange rates for an asset, it is assumed that the Stellar asset can be exchanged 1-for-1 with the corresponding off-chain asset.

Is that safe to assume? Are there anchors in the ecosystem operating that way?

Reply:
All existing anchors operate this way, since the Stellar asset they use is pegged to the fiat currency their users have.